target/linux/mediatek/dts/mt7981a-comfast-cf-e393ax.dts

```diff
@@ -216,6 +216,11 @@
 			label = "lan1";
 		};
 
+		port@1 {
+			reg = <1>;
```

Comment:
So this should be changed too: https://github.com/openwrt/openwrt/blob/main/target/linux/mediatek/filogic/base-files/etc/board.d/02_network#L57